### ISSUE TITLE
[Accessibility]A11y_VSCode_MenuBar_Keyboard: Item List of "MenuBar" is not accessible by the keyboard.     

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51966]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : MAC OS HighSierra10.13.5

**Additional Details:**
Keyboard,MAS2.1.1,MAS2.4.3

**Repro Steps:**
1.Navigate to "Menu Bar" at top of the screen after login.
2.Try to access the "MenuBar" list item such as "File,View,Edit".
3.Observed that neither keyboard shortcuts such as "Alt+F","Alt+V" nor keyboard keys such as "tab/enter/arrows" MenuBar is not accessible.

**Actual:**
Using keyboard shortcuts such as "Alt+F","Alt+V" menubar item is not accessible even using keyboard keys such as "tab/enter" the menubar is not accessible.
User when uses the mouse then only MenuBar item can be accessible.

**Expected:**
Without using mouse "MenuBar" can be accessible by using keyboard keys or shortcuts keys.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/fixit-details.aspx?fixitid=326](url)

**User Impact:**
If user is not able to access by the shortcuts or tab keys then user will not access all the actionable control of the application.

**MAS Reference:**
MAS2.1.1- [https://microsoft.sharepoint.com/teams/msenable/mas/pages/fixit-details.aspx?fixitid=326](url)
MAS2.4.3- [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}
](url)

**Attachment For Reference:**
[[Accessibility]A11y_VSCode_MenuBar_Keyboard.pptx](https://github.com/Microsoft/vscode/files/2105135/Accessibility.A11y_VSCode_MenuBar_Keyboard.pptx)
Does this issue occur when all extensions are disabled?: Yes

**Comment by sbatten** [comment]
resolved by new custom menubar
